compiler/onecc-docker/debian/postinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html
+# Boradly speaking, the `postinst` is called after a package is unpacked.
+
+set -e
+
+sudo chmod +x /usr/share/one/bin/onecc-docker
```

Comment:
Um, I suggest to use `dh_fixperms` function in rules file.
https://manpages.debian.org/testing/debhelper/dh_fixperms.1.en.html

ex)
```
override_dh_fixperms:                                                                               
  dh_fixperms                                                                                       
  chmod +x <files>
```

Reply:
@jyoungyun 
Oh, thank you so much for the code review. Thanks to you, I learned that it is possible to set permissions with debhelper. I did a simple action test after modifying the code, and I checked that the authority was well granted.